package.json

```diff
@@ -30,25 +30,23 @@
   "main": "src/index.pug",
   "repository": "github:tikui/tikui",
   "scripts": {
-    "serve": "concurrently npm:express:sass:cache npm:express:serve",
-    "build": "rimraf dist && concurrently npm:sass:build npm:pug:build npm:assets:build",
+    "serve": "node build/tikui.js serve",
```

Comment:
Do you think use alphabetical order for scripts is a good idea?
I'm crazy about alphabetical order :smirk:

Reply:
I think it's a good order because it's ordered by primary commands followed by secondaries then internal.

On Primary commands, I've sorted by usage:

1. serve to use tikui
2. build to make a dist
3. lint to check sources